release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1460,6 +1464,64 @@ metadata:
+[source.terminal]
++
+----
+[ERROR] : [OperatorImageCollector] pinging container registry registry.redhat.io: Get "http://registry.redhat.io/v2/": dial tcp 23.217.255.152:80: i/o timeout
+----
++
+With this update, the plugin checks the local cache during the disk-to-mirror process to determine the digest, avoiding the need to query the registry. (link:https://issues.redhat.com/browse/OCPBUGS-36214[*OCPBUGS-36214*])
+
+* Previously, when using oc-mirror plugin v2 in mirror-to-disk mode in disconnected environments, the plugin was unable to access `api.openshift.com` to download `graph.tar.gz`, resulting in mirroring failures. With this update, the plugin now searches the local cache for the graph image in disconnected environments where the `UPDATE_URL_OVERRIDE` environment variable is set. If the graph image is missing, the plugin skips it without failing. (link:https://issues.redhat.com/browse/OCPBUGS-38469[*OCPBUGS-38469*])
+
+* Previously, oc-mirror plugin v2 failed to mirror operator catalogs from disk-to-mirror in fully disconnected environments. This issue also affected catalogs that specified a `targetCatalog` in the `ImageSetConfiguration` file. With this update, the plugin can now successfully mirror catalogs in fully disconnected environments, and the `targetCatalog` functionality works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34521[*OCPBUGS-34521*])
+
+* Previously, with the oc-mirror plugin v2, there was no validation for the `-v2` vs `--v2` flags for the `oc-mirror` command. As a result, users who mistakenly used `-v2`, that sets the log level to 2 instead of `--v2`, which switches to oc-mirror plugin v2, causing unclear error messages. With this update, flag validation is provided. If `-v2` is used while the `ImageSetConfig` is using the `v2alpha1` API and `--v2` is not specified, an error message is displayed. The following message is now enabled that provides a clear guidance to the user:
++
+[source,terminal]
+----
+[ERROR]: Detected a v2 `ImageSetConfiguration`, please use `--v2` instead of `-v2`.
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGS-33121[*OCPBUGS-33121*])
+
+* Previously, oc-mirror plugin v2 did not automatically perform retries when it encountered errors on registries, such as timeouts, expired authentication tokens, HTTP 500 errors, and so on. With this update, retries for these errors are implemented, and users can configure retry behavior with the following flags:
++
+* `--retry-times`: Specifies the number of retry attempts. Default is 2.
+* `--retry-delay`: Sets the delay between retries. Default is 1 second.
+* `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
+* `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
```

Comment:
these list items should be level 2 so that they're nested under their respective RN:
```suggestion
** `--retry-times`: Specifies the number of retry attempts. Default is 2.
** `--retry-delay`: Sets the delay between retries. Default is 1 second.
** `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
** `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
```